src/components/templates/ExternalRoom/ExternalRoom.scss

```diff
@@ -1,41 +1,92 @@
 @import "scss/constants";
 
+$border-radius: 22px;
+$spacing: 20px;
+
 .ExternalRoom {
   height: 100%;
-  width: 80%;
   display: flex;
+  background-position: center;
+  background-size: cover;
   align-items: center;
   justify-content: center;
-  margin: 0 auto;
+  flex-direction: column;
 
   &__message {
-    padding: 60px;
     display: flex;
     flex-direction: column;
+    justify-content: center;
     align-items: center;
-    justify-content: space-evenly;
-    background: $light-grey;
-    box-shadow: box-shadow--large(0.5);
-    border-radius: $border-radius--xl;
+    margin: $spacing--xxl 0;
+    font-family: Rubik;
+    font-weight: $font-weight--300;
+    font-size: $font-size--xl;
   }
 
   &__content {
     display: flex;
     flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    margin-top: $spacing--xxl;
+    max-height: 80vh;
+    border-radius: 20px;
```

Comment:
your own var `$border-radius`

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2224/commits/7ec8c1269d6a24483177fb1e5bf785aa502864c9